.gitignore

```diff
@@ -32,7 +32,10 @@ _artifacts/
 e2e-test/image/e2e/chaos-mesh
 e2e-test/image/e2e/manifests/
 
-/pkg/uiserver/embedded_assets_handler.go
+/pkg/dashboard/uiserver/embedded_assets_handler.go
+
+pkg/dashboard/swaggerdocs/*
+!pkg/dashboard/swaggerdocs/placeholder.go
```

Comment:
@YangKeao @g1eny0ung Should we do that?

Reply:
no go code should be ignored. Ignoring go codes will drop dependencies in `go.sum` and `go.mod`.

Reply:
Discussed with @YangKeao , we need to keep all the go code, so here I am going to modify it.